pkg/controller/workload/workload_processor.go

```diff
@@ -208,53 +221,44 @@ func (p *Processor) deleteFrontendData(id uint32) error {
 }
 
 func (p *Processor) removeServiceResource(resources []string) error {
-	var err error
 	for _, name := range resources {
 		telemetry.DeleteServiceMetric(name)
-		p.ServiceCache.DeleteService(name)
-		if err = p.removeServiceResourceFromBpfMap(name); err != nil {
-			return err
-		}
+		svc := p.ServiceCache.DeleteService(name)
+		_ = p.removeServiceResourceFromBpfMap(svc, name)
```

Comment:
```c
_ = p.removeServiceResourceFromBpfMap(svc, name)
svc := p.ServiceCache.DeleteService(name)
```
If the sequence is changed, does DeleteService not need to be modified?

Reply:
no sequence is changed